compose/runtime/runtime/src/nativeMain/kotlin/androidx/compose/runtime/ActualNative.native.kt

```diff
@@ -59,19 +60,21 @@ actual class AtomicReference<V> actual constructor(value: V) {
 }
 
 internal actual class AtomicInt actual constructor(value: Int) {
-    private val delegate = kotlin.native.concurrent.AtomicInt(value)
+    private val delegate = kotlin.concurrent.AtomicInt(value)
     actual fun get(): Int = delegate.value
     actual fun set(value: Int) {
         delegate.value = value
     }
     actual fun add(amount: Int): Int = delegate.addAndGet(amount)
 }
 
+@OptIn(ExperimentalNativeApi::class)
```

Comment:
We should keep in mind that we can't release Compose for iOS if at that time it is still be experimental. It doesn't add binary compatibility guarantees, which we need for stable Compose targets.